Source/santad/DataLayer/WatchItemPolicy.h

```diff
@@ -33,12 +33,25 @@ namespace santa {
 // Forward declarations
 enum class WatchItemPathType;
 enum class WatchItemRuleType;
+template <typename T>
+struct SharedPtrValueHash;
+template <typename T>
+struct SharedPtrValueEqual;
```

Comment:
```suggestion
template <typename T> struct SharedPtrValueHash;
template <typename T> struct SharedPtrValueEqual;
```

Reply:
If clang-format doesn't like this we either need to modify its config or find some other way to make this clearer as it's hard when scanning down to see that this is 2 declarations instead of 4.

Reply:
Spoke offline. clang-format 19 looks to have what we want, but it's not currently supported in the github runners. Will reevaluate later. For now, regrouped things to hopefully be more clear.